docs/concepts.rst

```diff
@@ -27,40 +32,79 @@ Projects
+
+Datasets
+^^^^^^^^
+
+Each project may have one or more *datasets*: on-disk representations of record data that can be inserted into Salesforce orgs, and which can also be modified and re-captured during the evolution of the project. Datasets are stored in the ``datasets`` directory. Learn more about datasets in TODO: link Automating Data Operations.
 
-CumulusCI stores service authentication details in an encrypted file in your configuration directory.
+Robot Framework Browser Automation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CumulusCI integrates Robot Framework browser automation for end-to-end testing. Each project contains a ``robot`` directory, which stores the resources and test suites for the project's Robot Framework test suites. New CumulusCI projects start with a simple Robot test case that creates a Contact record. 
+
+While Robot Framework is used primarily for browser automation testing, in a small number of situations, Robot Framework may also be harnessed to help configure orgs where other strategies and APIs are insufficient.
+
+Unpackaged Metadata
+^^^^^^^^^^^^^^^^^^^
+
+In the Product Delivery Model, a product doesn't just encompass the contents of a managed package or a single deployment. It also includes *unpackaged metadata*: extra bundles of Salesforce metadata that further tailor an org or complete the product.
+
+In a CumulusCI project, all unpackaged metadata is stored in subdirectories within the ``unpackaged`` directory. Unpackaged metadata plays multiple roles within the Product Delivery Model, including preparing an org for installing packages, adding more customization after the package or application is deployed, and customizing specific orgs that are used in the product's development process.
+
+Learn more about managing unpackaged metadata in TODO: link Managing unpackaged configuration.
+
+Project Orgs & Services
+-----------------------
+
+Orgs and services are external, authenticated resources that each project uses. CumulusCI makes it easy to connect orgs, as well as services like a GitHub account or a MetaDeploy instance, to a single project or to use them across projects. 
 
 Orgs
-++++
+^^^^
+
+Each project has its own set of orgs, including active scratch orgs, persistent orgs like a production or packaging org, and predefined scratch org configurations. CumulusCI securely stores org authentication information in its keychain, making it easy to access connected orgs at any time. The ``cci org list`` command shows all of the orgs connected to a project.
 
-Each project's keychain stores authentication information for all of the orgs that are in use by the project, including scratch orgs, persistent orgs like a production or packaging org, and information about scratch orgs that are yet to be created.
+In most situations, orgs are connected only to a single project. However, CumulusCI also offers a global keychain, making it possible to share persistent orgs across multiple projects.
```

Comment:
This paragraph is true, but it's an advanced use case that can probably wait for the section on orgs.

Reply:
True. I included it to align with the reference to the global keychain in the Services section - both at least allude to the existence of both project-local and global keychains.

Reply:
I buy that. Change made.